rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -334,7 +334,23 @@ def test_find_uniswap_v2_lp_token_price(inquirer, ethereum_manager):
     address = '0xa2107FA5B38d9bbd2C461D6EDf11B11A50F6b974'
     identifier = ethaddress_to_identifier(address)
     inquirer.inject_evm_managers([(ChainID.ETHEREUM, ethereum_manager)])
-    price = inquirer.find_uniswap_v2_lp_price(EvmToken(identifier))
+    price = inquirer.find_lp_price(
+        token=EvmToken(identifier),
+        abi_name='UNISWAP_V2_LP',
```

Comment:
why do you use the abi here? the code doesn't use the abi argument when querying the price

Reply:
This was done because there is no protocol in the uniswap v2 lp tokens. As discussed in discord, price calculation for uniswap v2 lp tokens is done based on the underlying tokens of the lp token, not based on the protocol attribute. There is no need for the abi_name parameter and it is removed.